docs/usage.md

```diff
@@ -62,10 +62,48 @@ the cloned repository.
 evg-module-manager disable --module wiredtiger
 ```
 
+## Performing git actions
+
+Once a revision is found on the base repo, that revision can be used to perform
```

Comment:
This feels like it is jumping into details immediately. It would be nice to preface this section with a description of the use-case a user would use this sub-command for. There are 2 use-cases, that we are covering here: (1) I want to create a new branch to work off of (the checkout case), and (2) I want to update my branch with the latest changes (the rebase/merge case). It might be nice to have a different section for each of those use-cases.